vendor/k8s.io/kubernetes/openshift-kube-apiserver/kubeadmission/register.go

```diff
@@ -48,8 +49,8 @@ var (
 		"security.openshift.io/SCCExecRestrictions",
 	)
 
-	// AfterKubeAdmissionPlugins are the admission plugins to add after kube admission, before mutating webhooks
-	openshiftAdmissionPluginsForKube = []string{
+	// openshiftAdmissionPluginsForKubeBeforeMutating are the admission plugins to add after kube admission, before mutating webhooks
+	openshiftAdmissionPluginsForKubeBeforeMutating = []string{
```

Comment:
why BeforeMutaging?

Mutating and validating webhooks have the same chain and therefore the same order.

Reply:
I have no idea it was like this before https://github.com/openshift/origin/pull/23463/files#diff-1c04fb343e80ed3e40f957b7d96073d2R88

should we change it?

Reply:
I just changed the name :)

Reply:
makes sense